src/ScalarMessaging.ts

```diff
@@ -693,6 +720,141 @@ async function getOpenIdToken(event: MessageEvent<any>) {
     }
 }
 
+async function sendEvent(
+    event: MessageEvent<{
+        type: string,
+        state_key?: string,
+        content?: unknown,
```

Comment:
```suggestion
        content?: IContent,
```

Perhaps this would be better?